Arendal

arendalHub opened the pull request.

Commit:
COMMIT N°2 DU 25-07-2019

CRU des modules Utilisateurs, Profils, Sites, Zones, Sections et Employes opérationnels mais sans la suppression.

Commit:
Merge branch 'arendal' of https://github.com/owo-rookit/OWO-GPM into arendal

Commit:
Commit du 28-07-2019

Remplacement des boutons et évolution (back end) sur l'authetification et les modules de gestion du personnel.

BlackPowerC commented:
It not a good idea to add for versionning IDE-files or auto-generate files.

BlackPowerC commented:
Le mieux c'est de supprimer ces fichiers chez toi et de remettre à jour le code source sur ta branche.

Geedok committed:
Front end & Modélisation

Ajout de nouvelles vues pour la page de connexion, le menu modulaire et un tableau de bord. Modification, suppression et ajout de migrations.

Commit:
COMMIT DU 07-08-2019

Commit:
COMMIT DU 13-08-2019

Résolution des bugs sur les différents modules de la gestion du personnel (sites, sections et zones)
Ajout de controles et de masques de saisie et des messages de confirmation d'opérations (sites, sections, zones et employés)

Commit:
COMMIT DU 20-08-219

Mise en place de l'affectation partielle des employés

Commit:
Résolution des conflits qvec le répertoire distant

Commit:
Résolution de conflits de mise à jour

Commit:
Suite

Commit:
Commit du 03-09-2019 [Personnel v0.1]

Commit:
COMMIT DU 05-09-2019
